stores/Todo.store.ts

```diff
@@ -63,6 +65,12 @@ export class TodoStore {
     this.saveTodos();
   };
 
+  deleteTodo = (todoId: string) => {
+    this._todos = this._todos.filter((todo) => todo.id != todoId);
```

Comment:
you could use `findIndex` and `splice` so you mutate the array, instead of creating a new one with `filter` (Mobx favors mutable data)

Reply:
Cool, didn't know splice was a method.